addons/cloudnative-pg/disable

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+
+
+cnpg_delete_manifest() {
+	kubectl="$SNAP/microk8s-kubectl.wrapper"
+
+	"${SNAP_DATA}/bin/kubectl-cnpg" install generate | $kubectl delete -f - > /dev/null
+
+	echo "$?"
+}
+
+cnpg_enable() {
```

Comment:
Why do you call this "enable"? Shouldn't it be "disable"?

Reply:
typo? XD fixing! :D